networks/.gitattributes

```diff
@@ -0,0 +1,4 @@
+cosmoshub-3/cosmoshub3_for_zero_height.json filter=lfs diff=lfs merge=lfs -text
+cosmoshub-3/snapshot.json filter=lfs diff=lfs merge=lfs -text
+osmosis-1/pregenesis.json filter=lfs diff=lfs merge=lfs -text
+osmosis-1/genesis.json filter=lfs diff=lfs merge=lfs -text
```

Comment:
Do you know if .gitattributes work if placed in subdirectories like this?

If not, we'll find out post-merge lol